algorithms/qaoa/knapsack/qaoa_knapsack.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<p>The wrapper function <code>eval_constraint</code> is redundant. You can use the function-syntax flavor of assignment inline -</p><p> <code></code><code></p><p>within=lambda: assign(constraint(v), aux)</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/667/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>